keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+#include "ergodox.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```